src/components/CategoryFilter.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable react/prop-types */
+// import PropTypes from 'prop-types';
+// import { useSelector } from 'react-redux';
+import { connect } from 'react-redux';
+
+const CategoryFilter = (props) => {
+  const categories = [
+    'All',
+    'Action',
+    'Biography',
+    'History',
+    'Horror',
+    'Kids',
+    'Learning',
+    'Sci-Fi',
+  ];
+  return (
+    <form>
+      <select
+        id="category"
+        name="category"
+        onChange={(e) => {
+          props.handleFilterChange(e.target.value);
+        }}
+      >
+        {categories.map((category) => (
+          <option
+            key={category}
+            value={category}
+          >
+            {category}
+          </option>
+        ))}
+      </select>
+    </form>
+  );
+};
+
+// CategoryFilter.propTypes = {
+//   handleFilterChange: PropTypes.func.isRequired,
+// };
```

Comment:
- [ ] [OPTIONAL] It’s better to delete **commented piece of codes** like these. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))